packages/valory/skills/trader_abci/handlers.py

```diff
@@ -96,6 +98,11 @@ def agent_ids(self) -> List[int]:
         """Get the agent ids."""
         return json.loads(self.staking_synchronized_data.agent_ids)
 
+    @property
+    def funds_status(self) -> FundRequirements:
+        """Get the fund status."""
+        return self.context.shared_state[GET_FUNDS_STATUS_METHOD_NAME]()
```

Comment:
Won't have to maintain an extra variable this way

```suggestion
        return self.context.shared_state[
            FundsManagerBehaviour.get_funds_status.__name__
        ]()
```

Reply:
https://github.com/valory-xyz/trader/issues/588